playbooks/nfs.yml

```diff
@@ -21,6 +21,7 @@
     - name: End play early if no NFS is needed
       when:
         - not cifmw_edpm_deploy_nfs | default('false') | bool
```

Comment:
@rlobillo care to change this by this?
```yaml
 (not cifmw_edpm_deploy_nfs | default('false') | bool) or (cifmw_architecture_scenario is defined)
```
I know the solution is **far** from perfect, but it will do the trick for now till we reorganize the code.